GDJS/Runtime/spriteruntimeobject.js

```diff
@@ -5,53 +5,53 @@
  */
 
 /**
- * @typedef {Object} Point Represents a point in a frame
- * @property {number} x X position of the point
- * @property {number} y Y position of the point
+ * @typedef {Object} SpritePoint Represents a point in a coordinate system.
+ * @property {number | string} x X position of the point.
```

Comment:
X and Y position must always be numbers. If someone or the existing code is storing strings, that's a mistake and this should be fixed.
So let's keep `number`.

Putting string there will destroy any optimization made by the JS engine, and will expose us to very nasty bug. (just try to do "1" + 1 in a JavaScript console).

Reply:
OK that was because of the use of parseFloat

Reply:
Ah I see! You're right this is confusing.
I think the parseFloat could be simply useless and added at a time where it was not sure if the points x/y were number or strings. Because they should be stored as number in the project JSON, so we should be able to careful remove them.